config/webpack.config.js

```diff
@@ -53,6 +66,13 @@ module.exports = env => {
     pluginEntries
   );
 
+  var publicPath;
+  if (production) {
+    publicPath = process.env.ASSET_PATH || '/webpack/';
+  } else {
+    publicPath = process.env.ASSET_PATH || `http://${devServerHostname}:${devServerPort}/webpack/`;
+  }
```

Comment:
nitpick, can be written as:
```js
publicPath =  process.env.ASSET_PATH || (production ? '/webpack/' : `http://${devServerHostname}:${devServerPort}/webpack/`);
```

but why is it needed? (or maybe it should be a separate PR?)

Reply:
it needed for webpackChunkName,  otherwise on develop env the splitted js won't be reachable.

Reply:
Yeah, I didn't find any better way for that than what Amir has here.
I wonder if we can somehow guess `http` vs `https` from config. That would make it smoother. But I'm fine with the code as is if we can't find it.

Reply:
@amirfefer I found it! We can use `process.argv.includes('--https')`